src/createIdentity.ts

```diff
@@ -1,22 +1,30 @@
-import { Wallet } from 'ethers';
-import { createPrivateKey } from './createPrivateKey';
-import { publicKeyByPrivateKey } from './publicKeyByPrivateKey';
+import { SigningKey, concat, keccak256, randomBytes } from 'ethers';
+import { stripHexPrefix } from './util';
 
-export interface Identity {
-  privateKey: string;
-  publicKey: string;
-  address: string;
-}
+/**
+ * create a privateKey
+ * @return {string}
+ */
 
-export const createIdentity = (entropy?: Buffer): Identity => {
-  const privateKey = createPrivateKey(entropy);
-  const wallet = new Wallet(privateKey);
+export const createPrivateKey = () => {
```

Comment:
You can also look into passing an entropy as a `string` or `Uint8Array`. This will also be applied to `createIdentity`